src/CXXTests/CMakeLists.txt

```diff
@@ -2,8 +2,6 @@ get_property(SPHERAL_CXX_LIBS GLOBAL PROPERTY SPHERAL_CXX_LIBS)
 
 add_subdirectory(DeviceTestLib)
 
-if(ENABLE_TESTS)
-  blt_add_executable( NAME spheral_cuda_test
-                      SOURCES Spheral_CUDA_Test.cc
-                      DEPENDS_ON Spheral_CXX ${spheral_blt_cxx_depends})
-endif()
+blt_add_executable( NAME spheral_cuda_test
+                    SOURCES Spheral_CUDA_Test.cc
+                    DEPENDS_ON Spheral_CXX ${spheral_blt_cxx_depends})
```

Comment:
Do we want to remove the guard to build this target, this will always build now regardless of ENABLE_TESTS

Reply:
I removed the guard because I was the one that added it, we could add it back if we see fit. I added it when I added CXX_ONLY build to disable building tests because of this test. I don't remember what the issue was, but there was some error that came up when building CXX_ONLY with this test before.

Reply:
Let's keep it, if codes have an issue with this target for some reason it's easier to allow them to just turn it off.